client/src/app/modules/module-instance/components/fields/draggable-list/draggable-list.component.ts

```diff
@@ -0,0 +1,20 @@
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {FieldComponent, FieldData} from '../../field/field.component';
+
+@Component({
+  selector: 'jms-draggable-list',
+  templateUrl: './draggable-list.component.html',
+  styleUrls: ['./draggable-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DraggableListComponent extends FieldComponent<FieldData> {
+  drop(event: CdkDragDrop<string[]>) {
+    moveItemInArray(
+      this.cData['options'],
```

Comment:
The order of the options can be different on the control than it is in the options. So when the component instantiates we need to make sure that we reorder the initial list.